backend/docs/API_Specifications/facility-v2.yaml

```
@@ -0,0 +1,382 @@
+        '200':
+          description: Facility updated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Facility'
+        '400':
+          description: Invalid input data
+        '404':
+          description: Facility not found
+ 
+  /v2/facility/search:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)